lib/resources/dstu2/medication_statement.yaml

```diff
@@ -0,0 +1,259 @@
+                 {"resourceType": "Medication",
+                   "id": "2750067",
+                   "code": {
+                     "coding": [
+                       {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                        "code": "1191",
+                        "display": "Aspirin",
+                        "userSelected": false}],
+                     "text": "aspirin"}}]}
+  url: http://hl7.org/fhir/DSTU2/domainresource-definitions.html#DomainResource.contained
+
+- name: dateAsserted
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Date(/time) when the medication statement was asserted by the information source.
+  example: '{"dateAsserted: "2012-07-13T00:00:00Z"}'
+
+- name: dosage
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  description: How the medication is/was used by the patient.
+  example: |
+            {"dosage": [
+              {"text": "325 mg, Oral, q4hr, PRN: headache",
+               "timing": {
+                 "repeat": {
+                   "boundsPeriod": {
+                     "start": "2014-08-11T13:42:00.000-05:00"}},
+                 "code": {
+                   "text": "q4hr"}},
+               "asNeededCodeableConcept": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "25064002",
+                    "display": "headache (finding)",
+                    "userSelected": false}],
+               "text": "headache"},
+               "route": {
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "26643006",
+                    "display": "Oral route (qualifier value)",
+                    "userSelected": false}],
+                 "text": "Oral"},
+               "quantityQuantity": {
+                 "value": 325,
+                 "units": "mg",
+                 "system": "http://unitsofmeasure.org",
+                 "code": "mg"}}]}
+  children:
+
+  - name: text
+    required: 'No'
+    cardinality: 0..1
+    type: string
+    description: Free text dosage information as reported about a patient's medication use.
+    example: '{"text": "2, Oral, 2x/Wk, first dose 05/05/15 10:00:00 EDT"}'
+
+  - name: timing
+    required: 'No'
+    cardinality: 0..1
+    type: Timing
+    description: The timing schedule for giving the medication to the patient.
+    example: |
+               {"timing": {
+                 "repeat": {
+                   "boundsPeriod": {
+                     "start": "2014-11-03T14:38:00.000-05:00"}},
+                 "code": {
+                   "coding": [
+                     "system": "http://hl7.org/fhir/v3/vs/GTSAbbreviation",
+                     "code": "TID",
+                     "display": "TID"],
+                   "text": "TID"}}
+
+  - name: asNeeded[x]
+    required: 'No'
+    cardinality: 0..1
+    type: boolean|CodeableConcept
```

Comment:
Put spaces around the pipe: 'boolean | CodeableConcept'

The rendering logic preserves formatting so these look cramped when viewed on the rendered site